addon/components/polaris-spinner.js

```diff
@@ -71,8 +71,9 @@ export default Component.extend(SvgHandling, {
   /**
    * @private
    */
-  spinnerSource: computed('normalizedSize', function() {
-    return `polaris/spinner-${this.get('normalizedSize')}`;
+  spinnerSVG: computed('normalizedSize', function() {
+    let size = this.get('normalizedSize') === 'large' ? 'large' : 'small';
+    return `/@smile-io/ember-polaris/illustrations/spinner-${size}.svg`;
```

Comment:
let's make a map with explicit image sizes, ex

```js
const spinnerSVGSources = {
  small: '/@smile-io/ember-polaris/illustrations/spinner-small.svg'
  large: /@smile-io/ember-polaris/illustrations/spinner-large.svg
}
```

This is to avoid issues with fingerprinting